eventdata/parameter_sources/elasticlogs_kibana_source.py

```diff
@@ -77,6 +83,17 @@ def __init__(self, track, params, **kwargs):
         if 'index_pattern' in params.keys():
             self._index_pattern = params['index_pattern']
         
+        if 'ignore_throttled' in params.keys():
+            if params['ignore_throttled'] == False:
```

Comment:
Refactoring: Pythons conditionals are circuit breaking so it's safe to add this if to the line above like:

```
if 'ignore_throttled' in params and not params['ignore_throttled']:
    self.ignore_throttled = False
```

The existing code works fine, perhaps this refactoring though makes it simpler/cleaner.

(Note that to check for the existence of a key in Python3 it's enough `key in mydict`, so this can also be simplified in other places too, see [docs](https://docs.python.org/3/library/stdtypes.html#dict))